src/main/java/com/mcneilio/orcmaker/orcer/OrcerFactory.java

```diff
@@ -0,0 +1,34 @@
+package com.mcneilio.orcmaker.orcer;
+
+import com.mcneilio.orcmaker.orcer.json.*;
+
+public class OrcerFactory {
+
+  public static Orcer getOrcer(String type) {
+    return switch (type) {
+      case "date" -> DateOrcer.getInstance();
+      case "string" -> StringOrcer.getInstance();
+      case "boolean" -> BooleanOrcer.getInstance();
+      case "smallint" -> SmallIntOrcer.getInstance();
+      case "tinyint" -> TinyIntOrcer.getInstance();
+      case "int" -> IntOrcer.getInstance();
+      case "bigint" -> BigIntOrcer.getInstance();
+      case "decimal(" -> DecimalOrcer.getInstance(); // does the '(' belong in there?
```

Comment:
I think so, and actually that raises an issue. This and other types use `startsWith` not `equals`